extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php

```diff
@@ -284,6 +284,36 @@ static public function onArticleViewHeader( Article $article, bool &$outputDone,
 		return true;
 	}
 
+	/**
+	 * When #REDIRECT is used make a proper 301 redirect
+	 *
+	 * @param Title $title
+	 * @param WebRequest $request
+	 * @param $ignoreRedirect
+	 * @param $target
+	 * @param Article $article
+	 * @return bool
+	 */
+	static public function onInitializeArticleMaybeRedirect(
+		Title &$title, WebRequest &$request, &$ignoreRedirect, &$target, &$article
```

Comment:
Minor: Objects are (effectively) passed by reference in PHP, so no need for the `&` on `$title`, `$request`, and `$article`.